chat-widget/src/common/telemetry/loggers/ariaTelemetryLogger.ts

```diff
@@ -8,14 +8,30 @@ import { Constants, AriaTelemetryConstants, EnvironmentVersion } from "../../Con
 import { IChatSDKLogger } from "../interfaces/IChatSDKLogger";
 import { TelemetryManager } from "../TelemetryManager";
 
+const AWTDefaultConfiguration = {
+    collectorUri: "https://browser.pipe.aria.microsoft.com/Collector/3.0/",
+    cacheMemorySizeLimitInNumberOfEvents: 10000,
+    disableCookiesUsage: false,
+    canSendStatEvent: (eventName: string) => { return true; }, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
What is this being used for?

Reply:
I believe it's the callback to whether to send telemetry. It's the default configuration from Aria that was reused.